tagged-localbuild.xml

```diff
@@ -1,55 +1,57 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
   <remote name="NXP" fetch="https://github.com/NXPNFCProject/"/>
-  <remote name="aosp" fetch="https://android.googlesource.com/"
-          review="https://android-review.googlesource.com/"/>
+  <remote name="aosp" review="https://android-review.googlesource.com/" fetch=".."/>
```

Comment:
Please keep this line this same, there is no need to change it.

Reply:
Isn't it easier to keep the manifest as-generated-by repo without any changes?

Reply:
`fetch=".."` will break repo init because we host this manifest on github.com as opposed to google servers (originally spotted in https://github.com/mer-hybris/android/pull/132#discussion_r611542917)